axelrod/plot.py

```diff
@@ -112,7 +112,8 @@ def _sd_ordering(self):
     @property
     def _sdv_plot_dataset(self):
         ordering = self._sd_ordering
-        diffs = self.result_set.score_diffs
+        diffs = [[diff for repetition in diffs for diff in repetition]
```

Comment:
This is confusing, can we use different iteration variables?

Reply:
I picked these in an attempt to not be confusing but happy to take any suggestions.

Reply:
91a3ee8 renames them to what is hopefully less confusing but still happy to change further if you think it can be improved :+1: :)

Reply:
It's still a little confusing since player and opponent usually indicate instances of Player classes. But it's better than before in that I can see what it's doing now.